Dockerfile

```diff
@@ -51,7 +52,7 @@ WORKDIR /src/jupyterhub
 
 # Build client component packages (they will be copied into ./share and
 # packaged with the built wheel.)
-RUN python3 setup.py bdist_wheel
+RUN python3 -m build --wheel
```

Comment:
This change isn't necessary, but it matches the pypi build
https://github.com/jupyterhub/jupyterhub/blob/02dead8ab1ad53655bdb461d9d7f94b52f2ecbec/.github/workflows/release.yml#L51-L53